`npm run test` fails with ERR_REQUIRE_ESM error

Output:

```
➜  claude-playground git:(main) ✗ npm run test

> claude-playground@1.0.0 test
> npm run test --workspaces


> @josheverett/bullshit-detector@1.0.0 test
> vitest

failed to load config from /Users/josh/workspace/claude-playground/packages/bullshit-detector/vitest.config.ts

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯ Startup Error ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/josh/workspace/claude-playground/node_modules/vite/dist/node/index.js from /Users/josh/workspace/claude-playground/node_modules/vitest/dist/config.cjs not supported.
Instead change the require of index.js in /Users/josh/workspace/claude-playground/node_modules/vitest/dist/config.cjs to a dynamic import() which is available in all CommonJS modules.
    at _require.extensions.<computed> [as .js] (file:///Users/josh/workspace/claude-playground/node_modules/vite/dist/node/chunks/dep-eRCq8YxU.js:36755:9)
    at Object.<anonymous> (/Users/josh/workspace/claude-playground/node_modules/vitest/dist/config.cjs:5:12)
    at _require.extensions.<computed> [as .js] (file:///Users/josh/workspace/claude-playground/node_modules/vite/dist/node/chunks/dep-eRCq8YxU.js:36755:9) {
  code: 'ERR_REQUIRE_ESM'
}



npm ERR! Lifecycle script `test` failed with error: 
npm ERR! Error: command failed 
npm ERR!   in workspace: @josheverett/bullshit-detector@1.0.0 
npm ERR!   at location: /Users/josh/workspace/claude-playground/packages/bullshit-detector 
```

--- a/packages/bullshit-detector/package.json
+++ b/packages/bullshit-detector/package.json
@@ -2,8 +2,15 @@
   "name": "@josheverett/bullshit-detector",
   "version": "1.0.0",
   "description": "AI-powered fact-checking and bullshit detection for Node.js applications",
+  "type": "module",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "types": "./dist/index.d.ts"
+    }
+  },
   "files": [
     "dist"
   ],

--- a/packages/bullshit-detector/tsconfig.json
+++ b/packages/bullshit-detector/tsconfig.json
@@ -1,8 +1,15 @@
 {
   "extends": ["@tsconfig/node22/tsconfig.json"],
   "compilerOptions": {
+    "module": "ES2022",
+    "target": "ES2022",
+    "moduleResolution": "Node",
     "types": ["vitest/globals"],
     "isolatedModules": true,
-    "verbatimModuleSyntax": false
-  }
+    "verbatimModuleSyntax": false,
+    "outDir": "./dist",
+    "rootDir": "./src"
+  },
+  "include": ["src/**/*"],
+  "exclude": ["dist", "node_modules"]
 }

